nautobot/extras/tests/integration/test_plugin_banner.py

```diff
@@ -5,7 +5,7 @@
 from nautobot.utilities.testing.integration import SeleniumTestCase
 
 
-@skipIf("dummy_plugin" not in settings.PLUGINS, "dummy_plugin not in settings.PLUGINS")
+@skipIf("example_plugin" not in settings.PLUGINS, "example_plugin not in settings.PLUGINS")
```

Comment:
In my opinion, we actually should be able to remove all of these plugin `skipIf` checks, as the dummy/example plugin should always be part of the the test setup (`nautobot/core/tests/nautobot_config.py`).